policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/ApplicationConfiguration.java

```diff
@@ -56,9 +56,9 @@ public StdPolicyConfiguration policyConfiguration() {
 
     @Bean
     @ConditionalOnProperty(prefix = "population", name = "policy", havingValue = "std")
-    public StdPolicyCacheWarmerFactory policyCacheWarmerFactory() {
+    public StdPolicyPrepopulationFactory policyCacheWarmerFactory() {
         LOGGER.info("Standard Policy Cache Warmer Instantiated");
```

Comment:
```suggestion
        LOGGER.info("Standard Policy Prepopulation Factory Instantiated");
```

Reply:
Moved all this logging to the controller classes, that print the getClass when used